pkg/ctl/create/fargate.go

```diff
@@ -49,9 +58,6 @@ func configureCreateFargateProfileCmd(cmd *cmdutils.Cmd) *fargate.CreateOptions
 }
 
 func doCreateFargateProfile(cmd *cmdutils.Cmd, options *fargate.CreateOptions) error {
-	if err := cmdutils.NewCreateFargateProfileLoader(cmd, options).Load(); err != nil {
-		return err
-	}
 	ctl, err := cmd.NewCtl()
```

Comment:
Ideally, it would be nice if we could extract all CLI related things from these functions, so that they are independent from any UI. Not sure how feasible it is at this point... (Not for this PR, I only mention it because I know you are working on refactoring these things)

Reply:
Indeed 👍